### PR TITLE
Add the 2.17 docs builders.

### PR DESCRIPTION
**Pull request by ipanova**

### CODE REVIEW

File: ci/jjb/jobs/projects.yaml [hunk 1]
```diff
@@ -57,8 +59,10 @@
     name: pulp-dev-2-17
     build_version:
     - 2-master:
-        pulp_version: '2.17'
+        pulp_version: '2.18'
```
Comment (ipanova) [inline ci/jjb/jobs/projects.yaml]: @Ichimonji10 i am unsure about this line?

**Comment by Ichimonji10** [comment]
I just merged https://github.com/pulp/pulp-ci/pull/573. It prunes a bunch of oudated jobs, and updates still-relevant ones. This has the unfortunate side-effect of creating merge conflicts. Please rebase.

File: ci/jjb/jobs/projects.yaml [hunk 2]
```diff
@@ -124,6 +128,7 @@
     upgrade_pulp_version:
         - 2.15-beta
```
Comment (Ichimonji10) [inline ci/jjb/jobs/projects.yaml]: I think this line can be dropped. We (nominally) test whether it's possible to upgrade from the prior stable release to the current stable release.

File: ci/jjb/jobs/projects.yaml (hunk 1 above)
Comment (Ichimonji10) [reply ci/jjb/jobs/projects.yaml]: Yes, I think this is correct.

**Comment by ipanova** [comment]
@Ichimonji10 done!

**Comment by dkliban** [comment]
We no longer use this repository to build docs. I would recommend making a PR against pulp and adding 2-master.yml and adding 2.17-beta.yaml file.

**Comment by Ichimonji10** [comment]
@dkliban In that case, can we drop JJB job definitions?

**Comment by ipanova** [comment]
closing in favour of https://github.com/pulp/pulp/pull/3593

**Pull request closed without merging**